src/PrestaShopBundle/Form/Admin/ShopParameters/ProductPreferences/PaginationType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace PrestaShopBundle\Form\Admin\ShopParameters\ProductPreferences;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class generates "Pagination" form
+ * in "Configure > Shop Parameters > Product Settings" page.
+ */
+class PaginationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('products_per_page', IntegerType::class)
+            ->add('default_order_by', ChoiceType::class, [
+                'choices' => [
+                    'Product name' => 0,
```

Comment:
be careful, this need to be translated in every language, you probably need to use the translator here and rely on TranslatorAwareType form instead of AbstractType

Reply:
Actually i tried switching to another language and translations worked fine for these strings. I also checked other forms that a migrated and they implement choices the same way. Im wondering if the changes are necessary here?

Reply:
It's working for me too